Remotion/Web/Core/Web.Core.csproj

```diff
@@ -876,16 +873,8 @@
     </None>
     <None Include="Res\HTML\jquery-1.6.4.js" />
     <None Include="Res\HTML\jquery.iFrameShim.js" />
-    <None Include="Res\HTML\DatePicker.js" />
```

Comment:
dropping the JS files, probably best as a last, separate commit instead of the "Register output" commit. in theory, the best place would be when we move the files, but that's just too complicated if we want to try for "always runnable"